cmake/helpers.cmake

```diff
@@ -444,8 +445,10 @@ function(config_lib config_name description)
         if(${config_name}_DISABLED)
             unset(${config_name}_DISABLED CACHE)
             set(${config_name} ${has_lib} CACHE STRING "${description}" FORCE)
+            set(${config_name}_INCLUDES ${has_include} CACHE STRING "Additional include paths for ${config_name}" FORCE)
```

Comment:
Do we need `FORCE` here? We are not specifying `FORCE` line 451.

Reply:
Yep, we need to use a `FORCE` here. 
We apply `FORCE` here to ensure the config is transitioned out of a `DISABLED` state. This is specifically done since the config would've been previously `FORCE`d into a `DISABLED` state when the dependencies weren't met on L457-L459:
https://github.com/wiredtiger/wiredtiger/blob/d827b2c1476dc9c2539c1772f51166a978432396/cmake/helpers.cmake#L457-L459 
As for why we use a `FORCE` on L457-L459, we want to make sure the config enters a disabled state regardless of what ever user input is applied to the configuration option i.e if the given dependencies aren't met we want to make sure that configuration option is not available (which would mean overriding any previously set value)

Reply:
Understood, thanks!